habitat_baselines/common/rollout_storage.py

```diff
@@ -23,7 +24,9 @@ def __init__(
         action_space,
         recurrent_hidden_state_size,
         num_recurrent_layers=1,
+        action_shape: Optional[Tuple[int]] = None,
```

Comment:
This isn't right. Optional[Tuple[Int, Int]] maybe? Or do you eally intend it o be a singleton tuple?